.github/workflows/deploy.yml

```diff
@@ -0,0 +1,154 @@
+name: deploy
+
+on:
+  push:
+    branches:
+      - master
+
+  pull_request:
+
+jobs:
+  cargo_docs:
+    name: Cargo Docs
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions-rs/toolchain@v1
```

Comment:
this needs an `id`:
https://github.com/getsentry/sentry-rust/blob/f2612287c6b99389be3916fc08b6570ca4ba4710/.github/workflows/ci.yml#L27

because you reference it below via `${{ steps.toolchain.outputs.rustc_hash }}`